syntax = "proto3";
package app.protos;

import "app/protos/items.proto";
import "app/protos/types.proto";

option go_package = "protos/parser";


message UserPref {
  string key = 1;
  int32 priority = 2;
  oneof value {
    double numValue = 3;
    StringList listValue = 4;
  }
}

message ItemExtended {
  Item item = 1;
  repeated Characteristic chars = 2;
  double totalWeight = 3;
}